conf/upgrade.yaml

```diff
@@ -0,0 +1,53 @@
+UPGRADE:
+  FROM_VERSION: "6.8"
+  TO_VERSION: "6.9"
+  OS: "rhel7"
+  DISTRO: ""
+  CAPSULE_AK:
+    CAPSULE_AK_RHEL6: "capsule_rhel6"
+    CAPSULE_AK_RHEL7: "capsule_rhel7"
+    CAPSULE_AK_RHEL8: "capsule_rhel8"
+  CLIENT_AK:
+    CLIENT_AK_RHEL6: "clientak_rhel6"
+    CLIENT_AK_RHEL7: "clientak_rhel7"
+    CLIENT_AK_RHEL8: "clientak_rhel8"
+  USER_DEFINED_CAPSULE_AK: ""
+  BASE_URL: ""  # Satellite6 repos url
+  CAPSULE_URL: ""  # Capsule repos
+  PRODUCTS:
+    - "satellite"
+    - "capsule"
+    - "client"
+    - "longrun"
+    - "n-1"
+  DISTRIBUTION: "downstream"
+  FOREMAN_MAINTAIN_SATELLITE_UPGRADE: "true"
+  SATELLITE_HOSTNAME: ""
+  CAPSULE_HOSTNAME: ""
+  DOWNSTREAM_FM_UPGRADE: "false"
```

Comment:
what does this means?

Reply:
This means we are using non-beta foreman-maintain version and we are good to whitelist the repositories-validate, repositories-setup in the foreman upgrade.